README.md

```diff
@@ -318,6 +318,12 @@ If you're a **Nix** user, you can install ripgrep from
 $ nix-env --install ripgrep
 ```
 
+If you're a [Flox](https://www.flox.dev) user, you can install ripgrep as follows:
```

Comment:
Can you change this to `**Flox**` in order to be consistent with other methods?

Reply:
Sure!